docs/.vuepress/config.js

```diff
@@ -165,6 +164,7 @@ module.exports = {
             "local-oidc",
             "mtls",
             "nginx",
+            "synology",
```

Comment:
+ add redirect

Reply:
Good catch, added in https://github.com/pomerium/pomerium/pull/2406/commits/1a5aa5187b42110e46a5eba0edb2d9ace51da295